src/geo/gmaps/gmaps-layer-view-factory.js

```diff
@@ -0,0 +1,47 @@
+var cdb = require('cdb');
+var log = require('cdb.log');
+var GMapsBaseLayerView = require('./gmaps-base-layer-view');
+var LeafletWMSLayerView = require('../leaflet/leaflet-wms-layer-view');
+
+var GMapsLayerViewFactory = function () {};
+
+GMapsLayerViewFactory.prototype.createLayerView = function (layerModel, mapModel) {
+  var layerType = layerModel.get('type').toLowerCase();
+  var LayerViewClass;
+
+  if (layerType === 'tiled') {
```

Comment:
After the short offline talk on Fri and giving it a second thought, I think "the simplest possible solution" would be a simple map (as it was before), since these if-else's are just mapping type-->class anyway.